components/server/src/workspace/messagebus-integration.ts

```diff
@@ -117,20 +118,23 @@ export class MessageBusIntegration extends AbstractMessageBusIntegration {
         const listener = new HeadlessWorkspaceLogListener(this.messageBusHelper, callback, workspaceID);
         const cancellationTokenSource = new CancellationTokenSource()
         this.listen(listener, cancellationTokenSource.token);
+        increaseMessagebusTopicReads('headless_workspace_log')
```

Comment:
this should be the actual topic we're listening on ... same goes for the other two